website/backend/settings.py

```diff
@@ -141,11 +142,16 @@
 # From Django's perspective, this is the input location.
 STATICFILES_DIRS = []
 
-if DEBUG:
-    STATIC_HOST = 'http://localhost:8081'  # Use webpack-dev-server on port 8081
-    STATICFILES_DIR = []  # type: Sequence[str]
+if not DEBUG:
+    GS_CREDENTIALS = service_account.Credentials.from_service_account_file(
```

Comment:
Cant we explore using option of exporting `GOOGLE_APPLICATION_CREDENTIALS`  key in the `.env` so that GCS will handle authentication implicitly.

Reply:
Done @ja-odur